sdcm/cluster.py

```diff
@@ -377,6 +377,22 @@ def host_id(self):
                 return full_nodetool_status[data_center][self.ip_address]['host_id']
         raise AssertionError(f"Could not find the requested node {self.ip_address} in nodetool status")
 
+    @property
+    def db_node_instance_type(self) -> Optional[str]:
+        backend = self.parent_cluster.cluster_backend()
+        if backend in ("aws", "aws-siren"):
+            return self.parent_cluster.params.get("instance_type_db")
+        elif backend == "azure":
+            return self.parent_cluster.params.get('azure_instance_type_db')
+        elif backend in ("gce", "gce-siren"):
+            return self.parent_cluster.params.get("gce_instance_type_db")
+        elif backend == "docker":
+            return "docker"
+        else:
+            LOGGER.warning("Unrecognized backend type, defaulting to 'Unknown' for"
```

Comment:
i think better to use self.log, because in this case, the log message also will contain the node name

Reply:
Fixed